src/config-hardware/audio-mic.md

```diff
@@ -0,0 +1,67 @@
+# Audio & Microphone Settings
+
+This section describes selection and configuration of audio input and output devices. Set system volume, volume levels per application, output devices, and input devices in `Settings` ➞ `Sound`.
+
+Pop!\_OS 22.04 uses the [PipeWire](https://github.com/pop-os/pipewire) server to handle audio and video streams. Previous versions of Pop!\_OS use [PulseAudio](https://www.freedesktop.org/wiki/Software/PulseAudio/).
+
+---
+
+## PipeWire
+
+Pop!\_OS 22.04 and above uses the PipeWire sound server. A sound server is a low-level service that controls communication of audio streams between the software and hardware.
+
+## Setting Volume Levels
+
+Drag the slider under `Volume Levels` right or left to increase or decrease the volume level.
+
+![Sound Settings](/images/audio-mic/sound-settings.png)
+
+### Over-Amplification
+
+`Over-Amplification` allows you to raise the volume over 100%. This can result in loss of audio quality. As a best practice, it is recommended to increase the volume in the application's individual volume settings.
+
+#### Over-Amplification Disabled
+
+![Over Amp Disabled](/images/audio-mic/over-amp-disabled.png)
+
+#### Over-Amplification Enabled
+
+![Over Amp Enabled](/images/audio-mic/over-amp-enabled.png)
+
+### Set Volume Levels Per Application
+
+Adjust the sliders in the `Volume Levels` section for each application as desired.
+
+![Per Application](/images/audio-mic/per-application.png)
+
+## Choosing an Audio Output Device
+
+Navigate to `Settings` ➞ `Sound`. Click the dropdown menu next to `Output Device`. Select the desired output device.
```

Comment:
It may be worth mentioning that a computer's sound card might only make one of its outputs available at a time (i.e. on most of our laptops, you can't just switch between headphones and speakers, because the sound card only makes the headphone output available when headphones are plugged in.)